### PR TITLE
fix: fixed colour contrast issues with contributor profile

### LINKED ISSUE #2346: Fix colour contrast issues on contributor profile header and about section
Fix colour contrast issues on contributor profile

### PR DESCRIPTION
**Pull request by nickytonline**
## Description

Kicks off some site wide colour contrast fixes starting with the contributor profile. Not that colour contrast issues in the feed in the contributor profile will be sorted in a separte PR as that component is used elsewhere.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Relates to #2342 
Fixes #2346

## Mobile & Desktop Screenshots/Recordings

**Before**

CleanShot 2023-12-15 at 13 43 55

**After**

CleanShot 2023-12-15 at 13 42 52

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Review by jpmcb — APPROVED** [review]

File: components/molecules/ContributorProfileInfo/contributor-profile-info.tsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ const ContributorProfileInfo = ({
     <div className="flex flex-col gap-6">
       <div className="pb-6 border-b">
         <div className="flex gap-2 items-center mb-2">
-          <Title className="!text-2xl !text-light-slate-12" level={3}>
+          <Title className="!text-2xl " level={3}>
```
Comment (brandonroberts) [inline components/molecules/ContributorProfileInfo/contributor-profile-info.tsx:56]: nit(non-blocking): remove extra spaces

File: components/molecules/ContributorProfileInfo/contributor-profile-info.tsx [hunk 2]
```diff
@@ -62,16 +62,16 @@ const ContributorProfileInfo = ({
           {isConnected && (
             <>
               {displayLocalTime && (
-                <span className="flex text-light-slate-10 gap-2 items-center">
+                <span className="flex  gap-2 items-center">
                   <Tooltip content="Time zone">
-                    <FiClock className="text-light-slate-9" />
+                    <FiClock className="" />
```
Comment (brandonroberts) [inline components/molecules/ContributorProfileInfo/contributor-profile-info.tsx:67]: nit(non-blocking): remove empty className

**Review by brandonroberts — APPROVED** [review]

File: components/molecules/ContributorProfileInfo/contributor-profile-info.tsx (hunk 1 above)
Comment (nickytonline) [reply components/molecules/ContributorProfileInfo/contributor-profile-info.tsx:56]: Search and replace leaving some space 😅

**Pull request merged**